client/checkout/classic/index.js

```diff
@@ -303,6 +308,14 @@ jQuery( function ( $ ) {
 	 * @param {Object} paymentMethod Payment method object.
 	 */
 	const handleOrderPayment = ( $form, { id } ) => {
+		if (
```

Comment:
Looks like a repeated piece of code to me, can this function be separated in a `ùtil.js` file and used across? That way, we can isolate the function's behavior and write the test only once?